### PR TITLE
Correct answer html page

### PR DESCRIPTION
**Pull request by niccireinking**

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/UserLevelServlet.java [hunk 1]
```diff
@@ -163,14 +163,17 @@ public String stripAnswer(String userAnswer){
...
             if(firstWord.equals("AN") || firstWord.equals("A") || firstWord.equals("THE")){
                 result = userAnswer.substring(userAnswer.indexOf(' ') + 1);
+                result.trim();
+                return result;
             }
         }
-        
+        result = userAnswer;
         result = result.trim();
+        System.out.println("Result: " + result);
```
Comment (miaboloix) [inline src/main/java/com/google/codeu/servlets/UserLevelServlet.java]: make sure you delete print statements when you finish! (this is super nit picky so feel free to ignore for now)

File: src/main/webapp/css/level5.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#map {
```
Comment (miaboloix) [inline src/main/webapp/css/level5.css]: did the css thing fix it?

File: src/main/webapp/js/puzzle-loader.js [hunk 3]
```diff
@@ -7,6 +7,50 @@ const parameterUsername = urlParams.get('user');
 if (!parameterUsername) {
     window.location.replace('/');
 }
+
```
Comment (miaboloix) [inline src/main/webapp/js/puzzle-loader.js:10]: delete this comment if you are implementing the correct.html page & things

File: src/main/webapp/puzzle.html [hunk 4]
```diff
@@ -27,7 +27,10 @@ <h1 class="z-depth-2" id="puzzle-title">Puzzle</h1>
             <form id="answer-form" action="/user-level" method="POST" class="hidden">
                 <h3 class="enter-answer">Submit Your Answer Here:</h3>
                 <textarea name="answer" id="answer-input"></textarea>
-                <input class="submit-answer z-depth-1" type="submit" value="Submit Answer">
+                <input id = "submission" class="submit-answer z-depth-1" type="submit" value="Submit Answer">
+                <p id = "demo"></p>
+                <p id="error" hidden>Sorry that is incorrect. Try again!</p>
```
Comment (miaboloix) [inline src/main/webapp/puzzle.html]: is this necessary if we implement the html pages?

**Review by miaboloix — CHANGES_REQUESTED** [review]
- see comments!

File: src/main/java/com/google/codeu/servlets/UserLevelServlet.java (hunk 1 above)
Comment (niccireinking) [reply src/main/java/com/google/codeu/servlets/UserLevelServlet.java]: oh thank you I forgot about this one. I'll take it out!

File: src/main/webapp/css/level5.css (hunk 2 above)
Comment (niccireinking) [reply src/main/webapp/css/level5.css]: um I changed the css and it didnt work, so I changed it to async defer and it worked but there was an error so I tired it on my map html without async and it worked. I took the async out and back to how I normally had it but with the new css and it worked so I guess the css was the problem but idk haha

File: src/main/webapp/js/puzzle-loader.js (hunk 3 above)
Comment (niccireinking) [reply src/main/webapp/js/puzzle-loader.js:10]: ok, do I need to put that somewhere else?

Comment (niccireinking) [reply src/main/webapp/js/puzzle-loader.js:10]: oh nvm I see what you're telling me to delete

File: src/main/webapp/puzzle.html (hunk 4 above)
Comment (niccireinking) [reply src/main/webapp/puzzle.html]: no I need to delete that

**Review by miaboloix — APPROVED** [review]

**Pull request merged**